data/addons/pyramid_fullauth.yaml

```
@@ -0,0 +1,15 @@
+addon: true
+category:
+- authorization
+- authentication
+demoUrl: ''
+description: Pyramid fullauth’s provides full user registration and management
```

Grammar fix. No `'s` needed.

```suggestion
description: pyramid_fullauth provides full user registration and management
```

I'm not sure whether you want to name it consistently with the PyPI package name.